src/Pickles/Pickles/DirectoryCrawler/FeatureNodeFactory.cs

```diff
@@ -57,12 +58,7 @@ public INode Create(FileSystemInfoBase root, FileSystemInfoBase location)
             if (this.relevantFileDetector.IsFeatureFile(file))
             {
                 Feature feature = this.featureParser.Parse(file.FullName);
-                if (feature != null)
-                {
-                    return new FeatureNode(file, relativePathFromRoot, feature);
-                }
-
-                throw new InvalidOperationException("This feature file could not be read and will be excluded");
+                return feature != null ? new FeatureNode(file, relativePathFromRoot, feature) : null;
```

Comment:
Why did you remove the invalidoperationexception? I know that a feature can be null if it's excluded by the tags, but malformed files will fail silently now ...